### PR TITLE
Minipot: Nomad v1.6.2 -data-dir fix

### PR DESCRIPTION
**Pull request by Sidicer**
Hello.

While trying to test `bsdpot/potman` on my local machine I noticed that with default configuration `nomad` fails to start on `minipot` guest.

When manually running nomad start command from `/usr/local/etc/rc.d/nomad`  we get `-data-dir` is required.

I added the minor fix to this branch

### CODE REVIEW

File: bin/minipot-init [hunk 1]
```diff
@@ -60,17 +60,18 @@ fi
...
 sysrc nomad_user="root"
+sysrc nomad_dir="/var/tmp/nomad"
 sysrc nomad_env="PATH=/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/sbin:/bin"
-if [ -n "$netif" ]; then
-	sysrc nomad_args="-config=/usr/local/etc/nomad/minipot-server.hcl -network-interface=$netif"
-else
-	sysrc nomad_args="-config=/usr/local/etc/nomad/minipot-server.hcl"
-fi
+sysrc nomad_args="-config=/usr/local/etc/nomad/minipot-server.hcl -data-dir=\$\{nomad_dir\}"
 sysrc consul_enable="YES"
```
Comment (grembo) [inline bin/minipot-init]:
```suggestion
sysrc nomad_args="-config=/usr/local/etc/nomad/minipot-server.hcl -data-dir=/var/tmp/nomad"
[ -n "$netif" ] && sysrc nomad_args+=" -network-interface=$netif"
sysrc consul_enable="YES"
```

File: bin/minipot-init [hunk 2]
```diff
@@ -60,17 +60,18 @@ fi
...
 sysrc nomad_enable="YES"
 sysrc traefik_enable="YES"
 sysrc traefik_conf="/usr/local/etc/minipot-traefik.toml"
 
+if [ -n "$netif" ]; then
+	sysrc nomad_args+=" -network-interface=$netif"
+fi
+
```
Comment (grembo) [inline bin/minipot-init]:
```suggestion
```

**Review by grembo — CHANGES_REQUESTED** [review]

**Review by grembo — APPROVED** [review]
LGTM

@pizzamig Could you double check? The reason why this was necessary is in: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=272162#c7 (my concerns weren't taken into account, so we have to change our startup script). Leaving in nomad_dir is probably a good idea in terms of being robust, in case the script changes again.

**Comment by grembo** [comment]
@pizzamig ping

**Review by pizzamig — APPROVED** [review]
It looks good to me!

**Pull request merged**